guides/release/pages.yml

```
@@ -1,325 +1,325 @@
       url: "autotracking-in-depth"
-    - title: "Patterns for Components"
+    - title: "Patterns liés aux Composants"
       url: "patterns-for-components"
-    - title: "Patterns for Actions"
+    - title: "Patterns liés aux Actions"
       url: "patterns-for-actions"
-    - title: "Making API Requests"
+    - title: "Faire des Requêtes API"
       url: "making-api-requests"
-    - title: "Native Classes In-Depth"
+    - title: "Classes Natives"
       url: "native-classes-in-depth"
 
-- title: "Application Development"
+- title: "Développement d'une Application"
   url: "toc-heading_application-development"
   is_heading: true
-- title: "Application Concerns"
+- title: "À propos des Applications"
   url: "applications"
   pages:
-    - title: "Applications and Instances"
+    - title: "Applications et Instances"
       url: "index"
       isAdvanced: true
-    - title: "Dependency Injection"
+    - title: "Injection de Dépendences"
       url: "dependency-injection"
       isAdvanced: true
     - title: "Initializers"
       url: "initializers"
       isAdvanced: true
-    - title: "The Run Loop"
+    - title: "La Run Loop"
       url: "run-loop"
       isAdvanced: true
     - title: "Ember Engines"
       url: "ember-engines"
       isAdvanced: true
-- title: Accessibility
+- title: Accessibilité
   url: accessibility
   pages:
-    - title: "Intro to Accessibility"
+    - title: "Intro à l'Accessibilité"
       url: "index"
-    - title: "Application Considerations"
+    - title: "Dans l'Application"
       url: "application-considerations"
-    - title: "Page Template Considerations"
+    - title: "Dans les Templates des Pages"
       url: "page-template-considerations"
-    - title: "Component Considerations"
+    - title: "Dans les Composants"
       url: "components"
-    - title: "Testing Considerations"
+    - title: "Dans les Tests"
       url: "testing"
-    - title: "Learning Resources"
+    - title: "Ressources d'apprentissage"
       url: "learning-resources"
 - title: "Configuration"
   url: "configuring-ember"
   pages:
-    - title: "Configuring Your App"
+    - title: "Configurer une App"
       url: "index"
-    - title: "Configuring Ember CLI"
+    - title: "Configurer Ember CLI"
       url: "configuring-ember-cli"
-    - title: "Handling Deprecations"
+    - title: "Gérer les Deprecations"
       url: "handling-deprecations"
-    - title: "Disabling Prototype Extensions"
+    - title: "Désactiver les Prototype Extensions"
       url: "disabling-prototype-extensions"
-    - title: "Specifying the URL Type"
+    - title: "Spécifier le Type d'URL"
       url: "specifying-url-type"
-    - title: "Embedding Applications"
+    - title: " Applications Nestées"
       url: "embedding-applications"
     - title: "Feature Flags"
       url: "feature-flags"
       isAdvanced: true
     - title: "Optional Features"
       url: "optional-features"
       isAdvanced: true
-    - title: "Build targets"
+    - title: "Navigateurs Cibles"
```

Environments in which an application is expected to run, so I would say:

```suggestion
    - title: "Environnements Cibles"
```